scripts/run-framework/run-framework.py

```diff
@@ -133,6 +133,9 @@ def setup_arg_parsing():
                         help='Logging level (default=%(default)s). '
                              'Options: debug, info, warning, error, critical')
 
+    parser.add_argument('--no_compress', dest='no_compress', action='store_true',
```

Comment:
Or, have it on by default with `compress` as the name. I mainly tried to follow the previous convention with no_docker, etc.